communications/consumers/messages.py

```diff
@@ -63,15 +67,27 @@ async def receive(self, text_data=None, bytes_data=None):
                 self.room_group_name,
                 {
                     'type': 'chat_message',
-                    'message': message,
+                    'oid': message_entity.oid,
+                    'message': message_entity.content,
+                    'sender_id': message_entity.sender_id,
+                    'receiver_id': message_entity.receiver_id,
+                    'created_at': message_entity.created_at,
                 }
             )
         except ApplicationException as e:
             logger.error(f"Error processing received message: {e}", exc_info=True)
 
     async def chat_message(self, event):
-        message = event['message']
+        oid = event['oid']
+        message = cipher_suite.decrypt(event['message']).decode()
+        sender_id = event['sender_id']
```

Comment:
Adding error handling for decryption can help prevent issues if the encryption key is invalid or the message data is corrupted.

Reply:
Done